core/controllers/admin.py

```diff
@@ -963,6 +964,24 @@ def post(self):
             raise self.InvalidInputException('This app cannot send emails.')
 
 
+class MemoryCacheHandler(base.BaseHandler):
+    """Handler for the memory cache."""
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
+        caching_services.flush_memory_cache()
+        self.render_json({})
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        memory_stats = caching_services.get_memory_stats()
```

Comment:
Make a domain object for memory_stats

Reply:
Done.

Reply:
Change name to cache_stats, and function name to get_memory_cache_stats()